nixos/modules/services/networking/seafile.nix

```diff
@@ -200,32 +289,58 @@ in
           preStart = ''
             if [ ! -f "${seafRoot}/server-setup" ]; then
                 mkdir -p ${dataDir}/library-template
-                mkdir -p ${ccnetDir}/{GroupMgr,misc,OrgMgr,PeerMgr}
-                sqlite3 ${ccnetDir}/GroupMgr/groupmgr.db ".read ${cfg.seafilePackage}/share/seafile/sql/sqlite/groupmgr.sql"
-                sqlite3 ${ccnetDir}/misc/config.db ".read ${cfg.seafilePackage}/share/seafile/sql/sqlite/config.sql"
-                sqlite3 ${ccnetDir}/OrgMgr/orgmgr.db ".read ${cfg.seafilePackage}/share/seafile/sql/sqlite/org.sql"
-                sqlite3 ${ccnetDir}/PeerMgr/usermgr.db ".read ${cfg.seafilePackage}/share/seafile/sql/sqlite/user.sql"
-                sqlite3 ${dataDir}/seafile.db ".read ${cfg.seafilePackage}/share/seafile/sql/sqlite/seafile.sql"
-                echo "${cfg.seafilePackage.version}-sqlite" > "${seafRoot}"/server-setup
+                # Load schema on first install
+                ${pkgs.mysql-client}/bin/mysql --database=ccnet_db < ${seafile-server}/share/seafile/sql/mysql/ccnet.sql
+                ${pkgs.mysql-client}/bin/mysql --database=seafile_db < ${seafile-server}/share/seafile/sql/mysql/seafile.sql
```

Comment:
```suggestion
                ${lib.getExe pkgs.mysql-client} --database=ccnet_db < ${seafile-server}/share/seafile/sql/mysql/ccnet.sql
                ${lib.getExe pkgs.mysql-client} --database=seafile_db < ${seafile-server}/share/seafile/sql/mysql/seafile.sql
```

Reply:
mysql-client doesn't have `meta.mainProgram` set, but otherwise I switched to using `getExe`.